src/seedu/addressbook/data/person/Address.java

```diff
@@ -84,4 +86,18 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString(Printable... printables) {
```

Comment:
this is incorrect. the `getPrintableString` method in the interface doesn't take any arguments.